tasks/build-react.js

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import fg from 'fast-glob';
+import { buildPackage } from './ts-tools.js';
+
+const buildReactWrapper = async () => {
+    const files = await fg(['./react/**/!(*.d).ts']);
+    await buildPackage(files);
+};
+
+buildReactWrapper();
```

Comment:
I like that added files to this workflow was so straight forward. Makes me wonder if we should name `buildPackage` better for continued expansion of its use?

Reply:
+1